components/workflow.rst

```diff
@@ -94,6 +94,16 @@ you can retrieve a workflow from it and use it as follows::
     $workflow->can($blogPost, 'publish'); // True
     $workflow->getEnabledTransitions($blogPost); // $blogPost can perform transition "publish" or "reject"
 
+If you want to initiate your workflow, you can call ``getMarking``::
```

Comment:
i think writing this paragraph here does not makes sens
as above there is `// Consider that $blogPost is in place "draft" by default`

maybe a new section where we describe the initial marking logic?

Reply:
Done !